src/js/components/pages.js

```diff
@@ -78,12 +78,27 @@ class Page {
     ];
   }
 
-  renderProfileImage() {
+  /**
+   *
+   * @param {String} [imgType] - One of profile or thumbnail - [default: thumbnail]
+   */
+  renderProfileImage(imgType) {
     const elements = [];
+
     if (this.profileImage) {
       const imageUrl = this.profileImage.meta.download_url;
       const imageAlt = this.profileImage.title;
-      elements.push($(`<img src="${imageUrl}" alt="${imageAlt}"></img>`));
+
+      if (imgType === "profile") {
+        const $profileImage = $("<img />", {
+          src: imageUrl,
+          alt: imageAlt,
+          class: "image image--rounded",
+        });
```

Comment:
please clone rather than construct - in this case mostly for consistency with the usual way we use webflow components, but also to avoid surprises if webflow devs assume we're cloning.